frontend/src/quantise-data.js

```diff
@@ -16,7 +29,7 @@ function quantiseData (tubeData) {
     });
     console.log(quantisedData);
     // save quantisedData to localStorage
-    // localStorage.setItem('quantisedData', JSON.stringify(quantisedData, null, 2));
+    localStorage.setItem('quantisedData', JSON.stringify(quantisedData, null, 2));
```

Comment:
I am still sceptical about this line but keen to hear your thoughts on it - since we are calling this function every 30 seconds / whenever we make a call to the API, Im worried we're just filling up local storage with files, especially if user / us leaves the site running (I've just had it on in the background for about 45 minutes whilst having breakfast heh)